reactor-core/src/main/java/reactor/core/publisher/MonoPeek.java

```diff
@@ -102,7 +99,7 @@ public Runnable onCompleteCall() {
 	@Override
 	@Nullable
 	public Runnable onAfterTerminateCall() {
-		return onAfterTerminateCall;
```

Comment:
maybe mark this method as "unused" in the javadoc to remove any ambiguity (`FluxPeek` uses it, `MonoPeek` doesn't and favors `MonoPeekTerminal` instead)